### ISSUE TITLE
logits moves to -inf after a few steps resulting in nan loss and zero learning

### ISSUE THREAD

**Issue by DanielGeorgeMathew** [QingyongHu/RandLA-Net#237]
As the title says, I modified the code to train on my dataset but no learning happens since the model outputs -inf logit values after a few steps. Following is the output just for debugging: 
![Screenshot from 2022-09-08 12-40-05](https://user-images.githubusercontent.com/17219895/189178312-ca03b78f-df1a-4540-9608-cfd2ce39c436.png)
![Screenshot from 2022-09-08 12-39-42](https://user-images.githubusercontent.com/17219895/189178314-6b9e0d62-a25c-467b-84d0-8431ec1c96f6.png)

**Comment by DanielGeorgeMathew** [comment]
Removed normals from training and fixed the issue
